frontend/package.json

```diff
@@ -23,10 +23,12 @@
     },
     "dependencies": {
         "@ant-design/icons": "^4.7",
+        "@chakra-ui/icons": "^2.0.19",
+        "@chakra-ui/react-use-disclosure": "^2.0.8",
```

Comment:
Is it possible to add them as peer dependencies? Ideally we should import everything from Redpanda UI as it re-exports the entire chakra UI